src/test/java/com/exasol/glue/ittests/CleanupIT.java

```diff
@@ -153,6 +156,39 @@ void testJobAlwaysFailsJobEndCleanup() {
         assertThat(exception.getMessage(), containsString("Intentionally fails all tasks."));
     }
 
+    @Test
+    void testSinkSuccessJobEndCleanup() throws SQLException {
+        final Table table = schema.createTableBuilder("table_cleanup_save") //
+                .column("c_str", "VARCHAR(3)") //
+                .column("c_int", "DECIMAL(9,0)") //
+                .column("c_double", "DOUBLE") //
+                .column("c_bool", "BOOLEAN") //
+                .build();
+        getSampleDataset() //
+                .write() //
+                .mode("append") //
+                .format("exasol") //
+                .options(getDefaultOptions()) //
+                .option("table", table.getFullyQualifiedName()) //
+                .save();
+        final String query = "SELECT * FROM " + table.getFullyQualifiedName() + " ORDER BY \"c_int\" ASC";
+        try (final ResultSet result = connection.createStatement().executeQuery(query)) {
+            assertThat(result, table().row("str", 10, 3.14, true).row("abc", 20, 2.72, false).matches());
```

Comment:
How does this prove the cleanup worked?

Reply:
Good point, there is assertion in the afterEach method since Spark session should be closed to force it. I am going to move it into dedicated method so that assertion is visible in tests.